src/main/java/org/spongepowered/api/event/entity/living/LivingEvent.java

```diff
@@ -37,8 +38,5 @@
      *
      * @return The {@link Living} involved
      */
-    Living getLiving();
-
-    @Override
-    Living getEntity();
+    Entity getEntity();
```

Comment:
Uh, no you're incorrect try it again, also you need to keep the override annotation.